### ISSUE TITLE
clarification on the mechanism analysis section

### ISSUE THREAD

**Issue by nguyentr17** [kdu4108/context-vs-prior-finetuning#2]
Hi, I enjoy reading your paper and thanks for open-sourcing the code! I have a question regarding the part in which you identify important components for models fine-tuned using intent format. I see that the pair accuracy is about 90%. When you do the patching experiments, do you filter to only samples that are accurately generated given a content (90% of the dataset) or do you include all samples?

**Comment by jkminder** [comment]
Hi, thank you and sorry for the late answer!

Yes, we only do the interpretability analysis on samples for which the model can correctly predict both the ctx and the prior. 
The _[collect_data](https://github.com/kdu4108/context-vs-prior-finetuning/blob/e7ea61f35428bbc3d580f73f2591bb345da92b02/analysis/circuit_utils/utils.py#L190)_ /_[filter_for_true_pairs](https://github.com/kdu4108/context-vs-prior-finetuning/blob/e7ea61f35428bbc3d580f73f2591bb345da92b02/analysis/circuit_utils/utils.py#L149)_ function does this, which is called in _[get_data](https://github.com/kdu4108/context-vs-prior-finetuning/blob/e7ea61f35428bbc3d580f73f2591bb345da92b02/analysis/circuit_utils/decoding.py#L231)_.

Let us know if you have any other questions!
